### PR TITLE
[skip ci] Improve comments in milvus.proto

### PR DESCRIPTION
**Pull request by nameczz**
Signed-off-by: nameczz <zizhao.chen@zilliz.com>
Issues: #7449

### CODE REVIEW

File: internal/proto/milvus.proto [hunk 1]
```diff
@@ -268,32 +268,62 @@ message CreatePartitionRequest {
...
-  string collection_name = 3; // must
-  string partition_name = 4; // must
+  // The collection name in milvus
+  string collection_name = 3;
+  // The partition name you want to drop
+  string partition_name = 4; 
 }
 
+/*
+* Check partition exist in collection or not.
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:286]: Check if partition exist in collection or not.

File: internal/proto/milvus.proto [hunk 2]
```diff
@@ -268,32 +268,62 @@ message CreatePartitionRequest {
...
-  common.MsgBase base = 1; // must
+  // Not useful for now
+  common.MsgBase base = 1;
+  // Not useful for now
   string db_name = 2;
-  string collection_name = 3; // must
-  string partition_name = 4; // must
+  // The collection name in milvus
+  string collection_name = 3;
+  // The partition name you want check
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:295]: The partition name you want to check

**Review by longjiquan — CHANGES_REQUESTED** [review]

File: internal/proto/milvus.proto (hunk 1 above)
Comment (nameczz) [reply internal/proto/milvus.proto]: Sorry for fix this slowly, thanks for advice

**Pull request merged**